src/DynamoCore/DynamoCore.csproj

```diff
@@ -379,6 +379,9 @@ limitations under the License.
   <ItemGroup>
     <Resource Include="BuiltInAndOperators\BuiltIn.Migrations.xml" />
   </ItemGroup>
+  <ItemGroup>
+    <Content Include="images\logo.png" />
```

Comment:
@mmisol I suggest to not include the Nuget package logo inside the projects, what do you think @aparajit-pratap ?

Reply:
I can remove it for this PR, but as it gets added automatically when updating the package, it will eventually slip in.

Reply:
I'm thinking to change the icon of location on the nuspec file, so in the future, it won't be automatically added to the projects